herc/aurorabackend.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Yeah, I was hesitant about this too. If this was Java and we made our interfaces explicit I'd put it there, but it's not. I like the `__init__.py` idea (although I never look for code there!).